### PR TITLE
Add Headline Summary to review page

### PR DESCRIPTION
**Pull request by andrewHEguardian**
## What does this change?

Add a summary of the user's changes

## How to test
Navigate to `/switch` then click the 'Change to monthly + extras' button. Witness the new page navigated to.

## Images
image

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: client/components/mma/switch/SwitchReview.tsx [hunk 1]
```diff
@@ -51,6 +51,19 @@ const SwitchReview = () => {
...
 				<Stack space={3}>
+					<section>
+						<Heading sansSerif>Review change</Heading>
+						<p
+							css={css`
+								${textSans.medium()};
+							`}
+						>
+							You will now support us with {mainPlan.currency}
+							{threshold} every month, giving you exclusive
```
Comment (jamesmockett) [inline client/components/mma/switch/SwitchReview.tsx:62]: I guess we want to make 'month' dynamic as well, although I hardcoded it myself in the section below this 🙃  We could tackle that separately once we have all the content on the page though.

**Review by jamesmockett — DISMISSED** [review]
👍

File: client/components/mma/switch/SwitchReview.tsx (hunk 1 above)
Comment (andrewHEguardian) [reply client/components/mma/switch/SwitchReview.tsx]: Good spot, I've made the change now since it's easy enough

**Review by jamesmockett — APPROVED** [review]

**Comment by andrewHEguardian** [comment]
[MMA: Review Change Page - Headline Summary copy](https://trello.com/c/NMaNe1jI/3757-mma-review-change-page-headline-summary-copy)

**Pull request merged**